node/network/bitfield-distribution/Cargo.toml

```diff
@@ -21,9 +21,9 @@ sc-network = { git = "https://github.com/paritytech/substrate", branch = "master
 polkadot-subsystem = { package = "polkadot-node-subsystem", path = "../../subsystem", features = [ "test-helpers" ] }
 bitvec = { version = "0.17.4", default-features = false, features = ["alloc"] }
 sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
-parking_lot = "0.10.0"
+parking_lot = "0.11.0"
 maplit = "1.0.2"
-smol = "0.2.0"
+smol = "0.3.3"
```

Comment:
I think this one is unused too :)

Reply:
It's used now :> for the delays which utilize a `smol::Timer`

Reply:
strange, I only see `use smol_timeout::TimeoutExt;` in code